File: BE/health-service/src/main/java/com/s206/health/bodyinfo/dto/response/BodyInfoResponse.java
package com.s206.health.bodyinfo.dto.response;

import java.math.BigDecimal;
import java.time.LocalDate;
import java.time.LocalDateTime;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;

@NoArgsConstructor
@Getter
@AllArgsConstructor
@Builder
public class BodyInfoResponse {

	private Integer bodyId;
	private Integer userId;
	private BigDecimal height;
	private BigDecimal weight;

	private BigDecimal bodyFat;
	private BigDecimal muscleMass;
	private LocalDateTime createdAt;
	private LocalDate measurementDate;

	private BigDecimal bmi;
	private BigDecimal bmr;

}

File: FE/mobile/src/main/java/com/example/diaviseo/network/body/dto/res/BodyInfoResponse.kt
package com.example.diaviseo.network.body.dto.res

data class BodyInfoResponse(
    val bodyId: Int?,
    val userId: Int,
    val height: Double,
    val weight: Double,
    val bodyFat: Double,
    val muscleMass: Double,
    val createdAt: String?,
    val measurementDate: String?,
    val bmi: Double,
    val bmr: Double
)
